R/loadOutput.R

```diff
@@ -0,0 +1,74 @@
+#'@title parse process output into R environment
+#'
+#'@details a \code{geoknife} method for loading data into R from a completed processing request
+#'
+#'@param .Object a \code{\link{geoknife}} object with a successful processID. 
+#'(See \code{\link{checkProcess}}).
+#'@return list of timeseries values. 
+#'@docType methods
+#'@keywords methods
+#'@import XML
+#'@import RCurl
+#'@author Jordan S. Read
+#'@export
+#'@examples
+#'\dontrun{
+#'gk <- geoknife() # create geoknife object
+#'gk # print geoknife object
+#'
+#'linearRing = bufferPoint(c(-111.48,36.95))
+#'setFeature(gk) <-list(LinearRing=linearRing)
+#'
+#'setAlgorithm(gk) <- getAlgorithms(gk)[4] # feature weighted
+#'
+#'# set the post inputs for the processing dataset
+#'setProcessInputs(gk) <- list('DATASET_ID'='prcp',
+#'                             'DATASET_URI'='http://thredds.daac.ornl.gov/thredds/dodsC/daymet-agg/daymet-agg.ncml',
+#'                             'TIME_START'='2010-01-01T00:00:00Z',
+#'                             'TIME_END'='2010-01-03T00:00:00Z',
+#'                             'DELIMITER'='TAB')
+#'
+#'gk <- startProcess(gk)
+#'Sys.sleep(10) # give it some time to process
+#'loadOutput(gk) # load and print output
+#'}
+#'
+setGeneric(name="loadOutput",def=function(.Object){standardGeneric("loadOutput")})
+
+#'@aliases loadOutput
+setMethod(f = "loadOutput",signature="geoknife",
+  definition = function(.Object){
+            if (isSuccessful(.Object)){
+              output <- outputParse(.Object)
+              return(output)
+            } else {
+              stop('processing is incomplete or has failed. See checkProcess()')
```

Comment:
what are you thinking for that? I figured the automated stuff would happen with `isSuccessful()` or `checkProcess()` preceding this function. What would you like to have happen?